<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1405</width>
    <height>830</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="current_Xpos">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>20</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="Xposition_label">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="take_picture">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Camera</string>
    </property>
   </widget>
   <widget class="ImageView" name="Current_Image">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>861</width>
      <height>751</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="start_vid">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stop_vid">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="get_focus">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Focus</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="current_focus">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>120</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="focus_label">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>150</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Focus</string>
    </property>
   </widget>
   <widget class="QPushButton" name="get_exposure">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Exposure</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="current_exposure">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>70</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>100</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Exposure</string>
    </property>
   </widget>
   <widget class="QPushButton" name="inc_exposure">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>70</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dec_exposure">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>90</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="current_Ypos">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>60</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QTextBrowser" name="current_Zpos">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>100</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QTextBrowser" name="current_Piezo">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>140</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="Y_pos_label">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>60</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="Z_pos_label">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>100</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
   </widget>
   <widget class="QLabel" name="Piezo_pos">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>140</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PIEZO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="red_laser">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>640 &quot;Laser&quot;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="green_laser">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>100</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>561 &quot;Laser&quot;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="blue_laser">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>130</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>488 &quot;Laser&quot;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="purple_laser">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>405 &quot;Laser&quot;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="prop_browser">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Properties</string>
    </property>
   </widget>
   <widget class="QLabel" name="Auto_label">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>190</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>AutoFocus Control</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="crisp_status">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>210</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="idle_button">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>190</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>IDLE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="log_cal_button">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>190</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>LOG CAL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dither_button">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>190</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>DITHER</string>
    </property>
   </widget>
   <widget class="QPushButton" name="set_gain_button">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>190</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>SET GAIN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="lock_button">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>LOCK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="unlock_button">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>280</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>UNLOCK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reset_offset_button">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>220</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
     </font>
    </property>
    <property name="text">
     <string>RESET OFFSET</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="lock_status">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>310</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="LockStatus">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>340</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Lock Status</string>
    </property>
   </widget>
   <widget class="QLabel" name="obj_NAlabel">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>220</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>NA:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="obj_NA">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>220</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="LED_spinbox">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>220</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="value">
     <double>50.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="LED_label">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>220</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>LED Intensity</string>
    </property>
   </widget>
   <widget class="QLabel" name="signal_to_noise_label">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>250</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>S/N:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="signal_noise_ratio">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>250</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="home_button">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="x_step_val">
    <property name="geometry">
     <rect>
      <x>1081</x>
      <y>310</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>1.000000000000000</double>
    </property>
    <property name="maximum">
     <double>10000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>100.000000000000000</double>
    </property>
    <property name="value">
     <double>1500.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="y_step_val">
    <property name="geometry">
     <rect>
      <x>1191</x>
      <y>310</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>1.000000000000000</double>
    </property>
    <property name="maximum">
     <double>10000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>10.000000000000000</double>
    </property>
    <property name="value">
     <double>700.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="x_step_label">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>330</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>X Step (um)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>330</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Y Step (um)</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="max_x_array_val">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>400</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-50000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>55000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>10.000000000000000</double>
    </property>
    <property name="value">
     <double>14000.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>420</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Max X Range</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="max_y_array_val">
    <property name="geometry">
     <rect>
      <x>1181</x>
      <y>400</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>10.000000000000000</double>
    </property>
    <property name="maximum">
     <double>25000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>10.000000000000000</double>
    </property>
    <property name="value">
     <double>1750.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>420</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Max Y Range</string>
    </property>
   </widget>
   <widget class="QPushButton" name="map_surface_button">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>480</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Map Surface</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reset_piezo">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>600</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset Piezo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="center_xy">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>630</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>HWCenter X,Y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="define_home">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>630</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>DefineHome</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hw_center_x">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>660</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>HWCenterX</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hw_center_y">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>660</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>HWCenterY</string>
    </property>
   </widget>
   <widget class="QPushButton" name="get_xy_speed">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>570</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>GetXYSpeed</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="min_y_array_val">
    <property name="geometry">
     <rect>
      <x>1181</x>
      <y>360</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-25000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>25000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>10.000000000000000</double>
    </property>
    <property name="value">
     <double>-3200.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>380</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Min Y Range</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="min_x_array_val">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>360</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-55000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>55000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>10.000000000000000</double>
    </property>
    <property name="value">
     <double>-14000.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>380</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Min X Range</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Z_stack">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>360</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Get ZStack</string>
    </property>
   </widget>
   <widget class="QPushButton" name="z_map">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>510</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Z-Map</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start_pos_timer">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>520</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>StartPosTimer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="array_map">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>540</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>ArrayMap</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1405</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageView</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
